normative/dtpr-stickers
# Removes admin and refactors code

# Notion card link(s)
- [Cleanup / Refactor of codebase](https://www.notion.so/normative/aa53a8fa0019423f893ff54bfbbe7ba6?v=fe60cb629d424b06b75bbd35d13d56b8&p=6d20144d5c59471fbf61c49bea8e4ae5)
- [Remove Admin Interface from the project](https://www.notion.so/normative/aa53a8fa0019423f893ff54bfbbe7ba6?v=fe60cb629d424b06b75bbd35d13d56b8&p=ebd31034507e458085147bb90580234e)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Separate view component from business container that communicate with api to fetch data;
- [x] Upgrade all libraries and change those that were deprecated to correspondent ones;
- [x] Introduce code standards styles;
- [x] Increase type checking into components & containers
- [x] Improve Navigation between pages (configure and use react router properly)
- [x] Cache in memory Place information to improve rendering
- [x] Add linting configurations
- [x] Remove admin pages
- [x] Extracts QRCode, firebase and airtable communications into their own files 

# How to test it
- Run into your machine then navigate between pages using
  - `/places/<placeId>
  - `/sensors/<sensorId>
  - `/sensors/<sensorId>/print` 
  
Between place and sensor you are able to navigate directly from page links, it should be smooth and relatively fast due to caching of places and airtable
